cobc/parser.y

```diff
@@ -12257,15 +12257,14 @@ call_body:
 		char* s = (char *)CB_LITERAL ($3)->data;
 		if (strncasecmp("Java.", s, 5) == 0) {
 			call_conv = CB_CONV_JAVA;
-		}
-		/* Check for malformed Java method names */
-		char* class_and_method_name = s + 5;
-		if (strchr(class_and_method_name, '.') == NULL) {
-			cb_error_x ($3, _("malformed Java method name '%s', expected format 'Java.ClassName.methodName'"), s);
-		}
-		/* Check for unsupported Java method calls with parameters or return values */
-		if($7 != NULL || $8 != NULL) {
-			CB_PENDING("Java method calls with parameters or return values");
+			/* Check for malformed Java method names */
+			if (strchr(s + 5, '.') == NULL) {
+				cb_error_x ($3, _("malformed Java method name '%s', expected format 'Java.ClassName.methodName'"), s);
```

Comment:
@xevor11 Actually this should also be tested. You can probably copy/paste and adapt the test on missing method in `run_java.at`, and just omit the `.`.